<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Email del Socio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="EmailEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;ZonaID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zonaElimEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="placeholderText">
      <string>Escribe el identificador de la Zona</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aceptarElimBoton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 165);
border-radius:10px;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regresarElimBoton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 165);
border-radius:10px;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Regresar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailElimEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ffffff;&quot;&gt;Eliminar Socio de la Zona&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
